ccmlib/dse_cluster.py

```diff
@@ -18,10 +18,12 @@
 class DseCluster(Cluster):
 
     def __init__(self, path, name, partitioner=None, install_dir=None, create_directory=True, version=None, dse_username=None, dse_password=None, dse_credentials_file=None, opscenter=None, verbose=False):
-        if dse_credentials_file:
-            self.load_credentials_from_file(dse_credentials_file)
-        else:
+        self.dse_username = None
+        self.dse_password = None
+        self.load_credentials_from_file(dse_credentials_file)
+        if dse_username is not None:
             self.dse_username = dse_username
+        if dse_password is not None:
```

Comment:
I've improved this by logging warnings if either dse_username or password are not set:

```
Warning: No dse username detected, specify one using --dse-username or passing in a credentials file using --dse-credentials.
Warning: No dse password detected, specify one using --dse-password or passing in a credentials file using --dse-credentials.
```

The download will still be attempted (and thus the Invalid url error will appear), I figured i'd leave it as is in the event the download ever doesn't require authentication.